terraform/log-alert.tf

```diff
@@ -9,12 +9,15 @@ data "azurerm_key_vault" "main" {
 }
 
 resource "random_string" "random" {
-  length           = 12
+  count   = 3
+  length  = 6
+  special = false
 }
 
+
 resource "azurerm_key_vault_secret" "main" {
-  name         = "user-offboarding-webhook-filter"
-  value        = "user-offboarding-${random_string.random.result}"
+  name         = "user-offboarding-token"
+  value        = "${random_string.random[0].result}-${random_string.random[1].result}-${random_string.random[2].result}"
```

Comment:
should we be using random_password? https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/password

Reply:
yep, updated